internal/deviations/deviations.go

```diff
@@ -652,4 +659,6 @@ var (
 	interfaceRefConfigUnsupported = flag.Bool("deviation_interface_ref_config_unsupported", false, "Device does not support interface-ref configuration when applying features to interface")
 
 	storageComponentUnsupported = flag.Bool("deviation_storage_component_unsupported", false, "Set to true for device that does not support telemetry path /components/component/storage")
+
```

Comment:
this line `_ = flag.Bool("deviation_ntp_non_default_vrf", false, "Set to true if device does not support ntp nondefault vrf")` can be removed. Since all new deviations will be proto fields we no longer have to set them as flags.